### PR TITLE
x11-wm/awesome: version bump to 4.1

### PR DESCRIPTION
**Pull request by tomoyat1**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

A pull request for bumping x11-wm/awesome to version 4.1

Build log is [here](https://bpaste.net/show/289a5b720b23)
Bugzilla bug is [here](https://bugs.gentoo.org/show_bug.cgi?id=613386)

### CODE REVIEW

**Comment by mkszuba** [comment]
@tomoyat1 No problems from the technical side, it seems. By the way - given your interest in this package, would you perhaps be interested in becoming its co-maintainer? You can find some more details on how being a proxied maintainer works here: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers

**Comment by tomoyat1** [comment]
@mkszuba How would I apply to become one? It would be an honor to help maintain this package.

File: x11-wm/awesome/awesome-4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+)
+
+src_configure() {
+	local mycmakeargs=(
+		-DPREFIX="${EPREFIX}"/usr
+		-DSYSCONFDIR="${EPREFIX}"/etc
+		-DWITH_DBUS=$(usex dbus)
+		-DWITH_GENERATE_DOC=$(usex doc $(usex doc) n)
+	)
+	if [ $(usex luajit) = "yes" ]; then
```
Comment (SoapGentoo) [inline x11-wm/awesome/awesome-4.1.ebuild]: `if use luajt; then`

**Comment by mkszuba** [comment]
Sorry about the delay, life caught up with me. What you should do at this point is open a "Maintainership request: x11-wm/awesome" bug on Bugzilla; please see the link I have posted for more details on what that bug should look like. We will take it from there. Afterwards you will become the primary maintainer of x11-wm/awesome, although you will still have someone else push changes to the repository for you (that's what the "proxied" part of "proxied maintainer" means).

As for the PR at hand, looks like we have reached maintainer timeout again as soon as I have concluded the final checks.

**Comment by tomoyat1** [comment]
@mkszuba: Will do when I have time. Thank you.

**Comment by johu** [comment]
This PR looks obsolete, closing then.

**Pull request closed without merging**